### DISCUSSION TITLE
Is it possible to turn on the screen remotely?

### DISCUSSION THREAD

**Discussion by julianrinaldi** [Blackymas/NSPanel_HA_Blueprint#1170] [Q&A]
I was wondering if it's possible to turn the screen on remotely. Basically I'd like to have the screen turn on when motion is detected in the room where the panel is located. Is this possible?

**Comment by edwardtfn** [comment]
Yes, it is possible and there are many different ways to achieve that.

I would suggest you call the service `esphome.xxxxx_wake_up` from your automation. Something like this:

```yaml
service: esphome.xxxxx_wake_up
data:
  reset_timer: true
```

The parameter `reset_timer` will reset the timer to fallback to Home page. The Sleep and dim timers will be reset anyways, so y8u can call this service even when the panel is already awake, it will just reset the timers in the case, or will wake-up if the panel is sleeping.

**Reply by julianrinaldi** [reply]
Awesome, thank you.
